rex/crash_tracer/full_tracer.py

```diff
@@ -0,0 +1,38 @@
+import os
+import logging
+
+import archr
+from tracer import TinyCore
+
+from . import CrashTracer, add_options, remove_options
+
+l = logging.getLogger("rex.HalfwayTracer")
```

Comment:
oops... generally, we use `logging.getLogger(__name__)` in order to avoid this kind of error.

Reply:
fixed in d9b8beb